corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from lxml import etree as ET
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.models import Application, PreloadAction
+from corehq.apps.app_manager.util import save_xform
+from corehq.apps.app_manager.xform import XForm
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = "Migrate an app from case management in the app builder to form builder"
+
+    def handle(self, *args, **options):
+        app_id = args[0]
+        self.options = options
+        logger.info('migrating app'.format(app_id))
+        self.migrate_app(app_id)
+        logger.info('done')
+
+    def migrate_app(self, app_id):
+        app = Application.get(app_id)
+        modules = [m for m in app.modules if m.module_type == 'basic']
+
+        for module in modules:
+            forms = [f for f in module.forms if f.doc_type == 'Form']
+            for form in forms:
+                preload = form.actions.case_preload.preload
+                if preload:
+                    xform = XForm(form.source)
+                    xform.add_case_preloads(preload)
+                    save_xform(app, form, ET.tostring(xform.xml))
+                    form.actions.load_from_form = form.actions.case_preload
```

Comment:
Resolved offline: the issue was with my understanding of the name `load_from_form`, which I was mistakenly interpreting as "load properties into case from submitted form." It actually means something like "load case properties that came from the form (i.e., from form builder) into the form (at runtime)." Clearer naming would be nice.